pysteps/io/importers.py

```diff
@@ -314,33 +343,17 @@ def import_saf_crri(filename, **kwargs):
     else:
         metadata["threshold"] = np.nan
 
-    # cut a window, to be defined here and only here
-    if True:
-        ny, nx = precip.shape
-        nx1 = 440
-        nx2 = nx - 440
-        ny1 = 250
-        ny2 = ny
-        precip = precip[ny1:ny2, nx1:nx2]
-        x1, x2 = metadata['x1'], metadata['x2']
-        xpixelsize = metadata['xpixelsize']
-        x1n = x1 + xpixelsize * nx1
-        x2n = x1 + xpixelsize * (nx2 - 1)
-        metadata['x1'], metadata['x2'] = x1n, x2n
-        y1, y2 = metadata['y1'], metadata['y2']
-        ypixelsize = metadata['ypixelsize']
-        y1n = y2 - ypixelsize * (ny2 - 1)
-        y2n = y2 - ypixelsize * ny1
-        metadata['y1'], metadata['y2'] = y1n, y2n
-
     return precip, None, metadata
 
 
-def _import_saf_crri_data(filename):
+def _import_saf_crri_data(filename, idx_x=None, idx_y=None):
     ds_rainfall = netCDF4.Dataset(filename)
     if "crr_intensity" in ds_rainfall.variables.keys():
-        data = np.array(ds_rainfall.variables["crr_intensity"])
-        precipitation = np.where(data == 65535, np.nan, data)
+        if idx_x is not None:
+            data = np.array(ds_rainfall.variables["crr_intensity"][idx_y, idx_x])
+        else:
+            data = np.array(ds_rainfall.variables["crr_intensity"])        precipitation = np.where(data == 65535, np.nan, data)
+            precipitation = np.where(data == 65535, np.nan, data)
```

Comment:
I've quickly scanned through most of the changes, and I'm not sure if this is correct, but it may also be a result of reading through it online.
Shouldn't it be:

`if idx_x is not None:
    data = np.array(...)
else:
    data = np.array(...)

precipitation = np.where(...)`

Reply:
you're absolutely right, the error was fixed by a4352e8